sds_data_manager/lambda_code/IAlirtCode/ialirt_ingest.py

```diff
@@ -39,8 +39,9 @@ def lambda_handler(event, context):
 
     # TODO: item is temporary and will be replaced with actual packet data.
     item = {
+        "apid": 478,
         "met": 123,
-        "ingest_time": "2021-01-01T00:00:00Z",
+        "ingest_date": "2021-01-01T00:00:00Z",
```

Comment:
I'd leave it as "time" if you are going to include the time component, or if you want date, remove the "T00:00:00Z"